conf/core/ConfigOptions.asciidoc

```diff
@@ -2518,6 +2544,28 @@ If true, POI/Polygon conflation will compare addresses as one of the criteria fo
 If the data being conflated is known to have poor address data, disabling this option may speed up
 conflation runtime performance.
 
+=== poi.polygon.address.translate.to.english
+
+* Data Type: bool
+* Default Value: `false`
+
+If true, Hootenanny will attempt to translate to English the value of any tag that is part of an
+address before normalizing it (note that address normalization may involve some form of language
+translation even when this setting is ).  This can have significant impact on the runtime performance of conflation when enabled
+and should only be enabled if the source data is known to have non-English addresses.  The
+configuration option, 'language.translation.translator', controls which translator is used.
+
+=== poi.polygon.address.use.default.language.translation.only
+
+* Data Type: bool
+* Default Value: `true`
+
+If true, POI/Polygon conflation will use only libpostal for translating languages in addresses,
+which occurs during address normalization.  If false, and poi.polygon.translate.addresses.to.english
+is enabled, POI/Polygon conflation will use additional to English language translation before
+address normalization.  Disable this only if libpostal's language translation is not adequate for
+your language translation needs.
+
 === poi.polygon.auto.merge.many.poi.to.one.poly.matches
```

Comment:
These new English translation related options for poi/poly addresses are a little confusing.  Prior to these changes, all translation was done with Joshua.  libpostal attempts to translate to English when normalizing an address.  I still left in the option to pre-translate with Joshua in the cases where libpostal doesn't translate very well against certain datasets.  libpostal is trained on actual address data, so would be expected to do better and maybe eventually 'poi.polygon.address.use.default.language.translation.only' could be removed.